lib/pages/chat/chat_header.dart

```diff
@@ -38,7 +38,7 @@ class ChatHeader extends StatelessWidget {
               size: 38,
               imageUrl: isDirect ? avatars.first.link : icon,
               avatars: avatars,
-              stackSize: 26,
+              stackSize: 24,
```

Comment:
Why are we resizing here ?

Reply:
26 make it a little cut, so change to 24